### PR TITLE
Add Ruby VM Magic Dashboard

### PR DESCRIPTION
**Pull request by jeffkreeftmeijer**
This pull request adds a Ruby VM magic dashboard, which will be added to the next minor version of the Ruby integration.

### CODE REVIEW

File: dashboards/ruby_vm/main.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
+  "metric_keys": [
+    "total_allocated_objects"
+  ],
+  "dashboard": {
+    "title": "MRI",
```
Comment (jvanbaarsen) [inline dashboards/ruby_vm/main.json:6]: Isn't this title a bit too generic? What if we want to add another ruby related dashboard in the future?

Comment (thijsc) [reply dashboards/ruby_vm/main.json:6]: I think it makes sense. If we add more MRI metrics later we could add that here.

**Review by thijsc — APPROVED** [review]

**Review by luismiramirez — APPROVED** [review]

**Review by matsimitsu — APPROVED** [review]

**Review by unflxw — APPROVED** [review]

File: dashboards/ruby_vm/main.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      {
+        "title": "Total Allocated Objects",
+        "description": "The cumulative number of allocated objects",
+        "line_label": "%name%",
+        "display": "LINE",
+        "format": "number",
+        "draw_null_as_zero": true,
+        "metrics": [
+          {
+            "name": "total_allocated_objects",
```
Comment (tombruijn) [inline dashboards/ruby_vm/main.json:18]:
```suggestion
        "title": "Allocated Objects",
        "description": "The number of allocated objects between minutely measurements",
        "line_label": "%name%",
        "display": "LINE",
        "format": "number",
        "draw_null_as_zero": true,
        "metrics": [
          {
            "name": "allocated_objects",
```
I've made the suggestion in PR https://github.com/appsignal/appsignal-ruby/pull/863 to not store the totals, but the differences between measurements.

Comment (jeffkreeftmeijer) [reply dashboards/ruby_vm/main.json:18]: Makes sense!

**Pull request merged**